src/com/lilithsthrone/game/dialogue/utils/InventoryDialogue.java

```diff
@@ -1113,9 +1113,9 @@ public void effects(){
 								}
 								
 							} else if(index == 5) {
-								if(item.getEnchantmentItemType(null)==null) {
+								if(item.getEnchantmentItemType(null)==null || item.getItemTags().contains(com.lilithsthrone.game.inventory.ItemTag.UNENCHANTABLE)) {
```

Comment:
If you add _import com.lilithsthrone.game.inventory.ItemTag;_ (after import _com.lilithsthrone.game.inventory.InventorySlot;_ for example), you can shorten this to _item.getItemTags().contains(ItemTag.UNENCHANTABLE)_ which makes it easier to read.